### PR TITLE
ALKIS Liegenschaftskataster for Hessen

### PR DESCRIPTION
**Pull request by anikaweinmann**
Pipeline to create a tile index for the ALKIS Liegenschaftskataster for Hessen communities.

### CODE REVIEW

File: ALKIS/Liegenschaftskataster/HE/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Hessen ALKIS Liegenschaftskataster tile index
+
+The tile index for teh Hessen ALKIS Liegenschaftskataster can be created with
+[this script](HE_ALKIS_LK.py).
```
Comment (linakrisztian) [inline ALKIS/Liegenschaftskataster/HE/README.md:4]:
```suggestion
[this script](HE_ALKIS_LK_tindex.py).
```

**Review by linakrisztian — COMMENTED** [review]
Looks good to me, thanks. Only small question about Docker/Github workflow.

File: .github/workflows/tindex_dop_nw.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Update Tile Index
+name: Update DOP NW Tile Index
```
[the remark this answers is not in the document]
Comment (anikaweinmann) [reply .github/workflows/tindex_dop_nw.yml:1]: I forget to upload the ALKIS LK HE workflow

File: docker/Dockerfile [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+
+# installations for ALKIS LK HE tindex
+RUN pip3 install requests remotezip
+
+# copy scripts
 COPY . /src/tile-indices
 
 WORKDIR /src/tile-indices
 
-RUN python3 DOP/NW/openNRW_DOP_tindex.py
```
[the remark this answers is not in the document]
Comment (anikaweinmann) [reply docker/Dockerfile:11]: It is replaced in the workflow for the HE ALKIS LK data

File: .github/workflows/tindex_alkis_lk_he.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+on:
+  # push:
+  #   branches:
+  #     - dop_nw
+  schedule:
+    # run tile index creation every month
+    - cron:  '0 1 6 * *'
+
+jobs:
+  update-tindex-dop-nw:
```
Comment (linakrisztian) [inline .github/workflows/tindex_alkis_lk_he.yml:12]:
```suggestion
  update-tindex-alkis-lk-he:
```

File: docker/Dockerfile (hunk 3 above)
[the remark this answers is not in the document]
Comment (linakrisztian) [reply docker/Dockerfile:11]: Ah ok. But didn't it need to be replaced back to NW DOP afterwards (for next iteration/month)?

**Comment by anikaweinmann** [comment]
@linakrisztian no it is not changed in the repo only in the Dockerfile inside of the pipeline

**Comment by linakrisztian** [comment]
> @linakrisztian no it is not changed in the repo only in the Dockerfile inside of the pipeline
But shouldn't then, this be contained in the Dockerfile (and the ALKIS version will be changed within pipeline)?
`RUN python3 DOP/NW/openNRW_DOP_tindex.py`

**Pull request merged**